cmd/rbac-manager/main.go

```diff
@@ -67,14 +66,27 @@ func main() {
 		)
 	}
 
-	mgr, err := manager.New(config.GetConfigOrDie(), manager.Options{})
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Scheme:             scheme,
+		MetricsBindAddress: fmt.Sprintf("%s:%d", commonOpts.MetricAddress, commonOpts.MetricPort),
+		Port:               9443,
+		LeaderElection:     commonOpts.ManagerLeaderElection,
+		LeaderElectionID:   "vault.crds.gocardless.com",
```

Comment:
```suggestion
		LeaderElectionID:   "rbac.crds.gocardless.com",
```

Reply:
We aren't using leader election at the moment, so this won't be a problem for the time being